app/javascript/components/widgets/components/widget-settings/widget-settings-styles.scss

```diff
@@ -1,54 +1,67 @@
 @import '~styles/settings.scss';
 
 .c-widget-settings {
-  width: rem(250px);
+  width: rem(310px);
```

Comment:
Can we make this 290px? As for iphone 5 screen width is 320px and we have a standard padding of rem(15px) each side?